### PR TITLE
chore(developer): consolidate external links in Developer messages

### PR DESCRIPTION
**Pull request by mcdurdin**
Establishes keyman-urls.ts as a single place for any links to external websites.

I searched all .ts files in Keyman Developer source and replaced references to websites with function calls here. Note that the names of individual functions within this class can be easily changed as needed, as the hard part was the consolidation.

@keymanapp-test-bot skip

### CODE REVIEW

File: developer/src/common/web/utils/src/keyman-urls.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  // TODO: Not formatting code here to avoid dep on @keymanapp/common-types for developer-utils
+  static readonly COMPILER_ERROR_CODE = (code: string) => `https://kmn.sh/${code}`; // code should be km##### (already formatted)
+
+  // help.keyman.com
+  static readonly VIRTUAL_KEYS = ()         => `${KeymanUrls.HELP_KEYMAN_COM}/developer/language/guide/virtual-keys#common-virtual-key-codes`;
+  static readonly FILE_TYPE = (ext: string) => `${KeymanUrls.HELP_KEYMAN_COM}/developer/current-version/reference/file-types/${ext}`;
+  static readonly KMN_REF = (topic: string) => `${KeymanUrls.HELP_KEYMAN_COM}/developer/language/reference/${topic}`;
+  static readonly HELP_KEYBOARD = (id: string)   => `${KeymanUrls.HELP_KEYMAN_COM}/keyboard/${id}`;
+  static readonly HELP_MODEL = (id: string)      => `${KeymanUrls.HELP_KEYMAN_COM}/model/${id}`;
```
Comment (darcywong00) [inline developer/src/common/web/utils/src/keyman-urls.ts:17]: Is /model a TODO?

Comment (mcdurdin) [reply developer/src/common/web/utils/src/keyman-urls.ts:17]: I don't understand the question?

**Review by darcywong00 — APPROVED** [review]
lgtm

File: developer/src/common/web/utils/src/keyman-urls.ts (hunk 1 above)
Comment (darcywong00) [reply developer/src/common/web/utils/src/keyman-urls.ts:17]: I thought help documentation for models aren't on help.keyman.com yet.

Comment (mcdurdin) [reply developer/src/common/web/utils/src/keyman-urls.ts:17]: Gotcha. The link was already there in .model_info files, so this doesn't change anything from Developer side. The TODO remains somewhere, doesn't need to be tracked here.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.282-beta](https://keyman.com/downloads/releases/beta/17.0.282)

**Pull request merged**